components/RatingsTable/index.tsx

```diff
@@ -170,9 +180,12 @@ Props): JSX.Element {
             <TableBody {...getTableBodyProps()}>
               {hasData && pagination
                 ? page.map((row, i) => {
-                    prepareRow(row);
+                  prepareRow(row);
+                  const info = getPlayerInfo(row.cells[0]['value']);
+                  const url = '/' + info[0] + '/player/' + info[1];
 
-                    return (
+                  return (
+                    <Link href={url} key={i}>
```

Comment:
link the entire row

Reply:
I can see this being an issue for users.
For instance, what if somebody is using mobile and wants to scroll down but accidentally keeps on clicking a link to a player page. We should only wrap the player's name with this. 

We should be adding the link in the column array for this table. There is an example of this in the files for the standings tables.

Let's change all these links to be only wrapping the player's name.

Reply:
corrected